### PR TITLE
Change chatty log stmt from INFO to DEBUG to ease prod log surfing

### PR DESCRIPTION
**Pull request by shdunning**
To help with Kibana storage limitations, change this one log statement to DEBUG so it doesn't end up in Kibana.

### CODE REVIEW

**Comment by jharting** [comment]
@shdunning at this point where inventory barely logs anything I am hesitant to remove one of the few log statements that works. Are you sure this solves a real problem?

**Comment by shdunning** [comment]
@jharting i doubt it will resolve the issue we're trying to track down with platform-tools, i was just questioning the value of this particular log statement. you would know better than i if it's useful in practice so feel free to do the right thing with this PR (nuke it if need be).

File: app/queue/ingress.py [hunk 1]
```diff
@@ -104,9 +104,9 @@ def add_host(host_data):
             ).inc()  # created vs updated
             # log all the incoming host data except facts and system_profile b/c they can be quite large
             logger.info(
-                "Host %s: %s",
+                "Host %s",
                 add_results.name,
-                {i: output_host[i] for i in output_host if i not in ("facts", "system_profile")},
+                extra={"host": {i: output_host[i] for i in output_host if i not in ("facts", "system_profile")}},
```
Comment (shdunning) [inline app/queue/ingress.py]: whoa @jharting was this the fix to get unstructured data to show up in Kibana?

Comment (jharting) [reply app/queue/ingress.py]: Yes I think this fixed it

**Pull request merged**